Summer 2020 Updates

Daehoon7992 opened the pull request:
0. Removed all files in www folder

Ui.r
1.  Removed unnecessary libraries - shinyjs, formattable and Added 'boastUtils'

2. Reformat the UI in terms of BOAST css style

3. Removed unnecessary css

4. Added arguments on functions

5. Added reference page and references

6. Added Alt text for 'Boxplot' and 'Histogram'

Server.r
1. Changed the id for readability

2. Removed 'background' argument on renderPlot function, since it causes error - unused argument

3. Streamed lines and removed white spaces.

Daehoon7992 committed:
Updates on ui.r and server.r

0. Removed all files in www folder

Ui.r
1.  Removed unnecessary libraries - shinyjs, formattable and Added 'boastUtils'

2. Reformat the UI in terms of BOAST css style

3. Removed unnecessary css

4. Added arguments on functions

5. Added reference page and references

6. Added Alt text for 'Boxplot' and 'Histogram'

Server.r
1. Changed the id for readability

2. Removed 'background' argument on renderPlot function, since it causes error - unused argument

3. Streamed lines and removed white spaces.

Daehoon7992 committed:
Digged into code and added comments by line

Ui.r
1. Added Alt text comment

2. change outputID name

Server.r
1. Removed unnecessary lines
 - hhh, hhh1, hhh2, ...

2. Changed the variable names - round2 to dataset, table1 to dataset2

3. Changed the Black dot(outlier) size from 2.2 to 2

4. Removed argument 'spacing' in output$values and output$values2, since it doesn`t do anything

5. Changed the width for mean and sd table from '33%' to '100%'

6. Added comments by line

Daehoon7992 committed:
Added width argument on plotOutput

Added width argument on plotOutput

neilhatfield reviewed (changes requested):
Changes to be made:

1. Remove lines 7-14 from the ui.R. This will necessitate the following change
2. Replace all `sliderInput2` calls with just `sliderInput`. You'll also have to remove the `from_min` and `from_max` arguments.
3. Remove the colons from the labels on the sliders
4. If there is only one column, you don't need to use `fluidRow` nor `column`; see lines 122-134 of ui.R
5. Why does the histogram get a popover but the box plot doesn't?
6. The font sizes in the plots are small. Ideally these would be in ggplot
7. Make sure that the axes of the histogram extend beyond the bars. Right now there are times where the bars actually go beyond the axes.
8. In line 168 of the ui.R, change the that to be `h3("Sample Statistics")`
9. Let's put all of these values into a single data table instead of two and use the `DT` package.

Daehoon7992 committed:
Updates for both ui.r and server.r from Neil`s feedback

ui.r
1. Removed sliderInput2 function and replaced sliderInput2 to sliderInput

2. Removed the colons on sliderInput

3. Changed strong("Sample Statistics") to h3("Sample Statistics")

4. Added space after commas

5. Removed bsPopover for histogram since I added the legend

6. Added the reference for library(DT) and library(ggplot2)

server.r
1. Removed output$value2 and combined the five number values into one dataset: output$value

2. Added the space after commas

3. Replaced the data table of statistics summary to DT table

4.  Replaced the plot into ggplot

#issues
1) some points appear on boxplot
2) might need to change the color in the plot?

Daehoon7992 committed:
Revisions for some wordings and design

ui.r
1. Changed the word 'Effect of Outlier' to 'Effects of an Outlier'

2. Revised some instructions

3. Changed the word 'Sample Statistics' to 'Summary Statistics for the Sample'

server.r
1. Added the width element on sendSweetAlert function

2. Changed some words on Y-axis of the plots

3. Removed the caption element and added 'dom' element, from Dr. Dennis`s feedback

neilhatfield committed:
Moved to app.R

Moved to the app.R format and made a few tweaks to plots and table
